modules/cms/rds.tf

```diff
@@ -0,0 +1,94 @@
+#
+# Externalize RDS.
+#
+# You will need to set AWS SSM parameters before you can provision RDS, like so:
+#   aws ssm put-parameter \
+#     --name circleci-REPLACEME_WITH_STACKNAME-rds_admin_user \
+#     --value REPLACEME_WITH_USERNAME \
+#     --type String
+#   aws ssm put-parameter \
+#     --name circleci-REPLACEME_WITH_STACKNAME-rds_admin_pass \
+#     --value REPLACEME_WITH_PASSWORD \
+#     --type String
+#
+
+#
+# Define postgres ingress. This only allows access in from the ecs tasks
+#
+resource "aws_security_group" "postgres_ingress" {
+  name        = "${local.prefix}-postgres-ingress"
+  description = "allow inbound access from ECS tasks only"
+  vpc_id      = "${module.network.vpc_id}"
+  tags        = "${module.tags.application_tags}"
+
+  ingress {
+    protocol        = "tcp"
+    from_port       = "5432"
+    to_port         = "5432"
+    security_groups = ["${module.circleci.circleci_services_sg_id}"]
+  }
+}
+
+module "rds_postgres" {
+  source = "terraform-aws-modules/rds/aws"
+
+  name              = "${local.prefix}-db"
+  identifier        = "${local.prefix}-db"
+  engine            = "postgres"
+  engine_version    = "10.6"
+  instance_class    = "${var.rds_instance}"
+  allocated_storage = "${var.rds_allocated_storage}"
+  storage_encrypted = true
+
+  # uncomment to use specific/non-default key
+  # kms_key_id        = "arm:aws:kms:<region>:<account id>:key/<kms key id>"
+
+  # NOTE: Do NOT use 'user' as the value for 'username' as it throws:
+  # "Error creating DB Instance: InvalidParameterValue: MasterUsername
+  # user cannot be used as it is a reserved word used by the engine"
+  username = "${data.aws_ssm_parameter.rds_admin_user.name}"
```

Comment:
formatting

Reply:
Interestingly, autoformatting (both in vscode and manually running `terraform fmt`) formats this line this way without any indent. Though I did find some formatting inconsistencies in variables.tf, and have committed those.